.github/workflows/app-deploy.yml

```diff
@@ -0,0 +1,15 @@
+name: "Front End Deploy"
+on:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  next-lint:
+    name: "Build"
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      ## Dummy step to clear up spurious pipeline failures
+      - name: Dummy
+        run: echo 'true'
```

Comment:
I don't understand what this does

Reply:
I added a blank app-deploy.yml so I could trigger workflows from my feature branches and test deploys. It works fine, but it is throwing errors in gitlabs validation because an empty file is not valid. This just makes that file valid so gitlab will stop complaining about an empty workflow file.

Here are the errors it is throwing: https://github.com/coforma/swift-tech-challenge/actions/workflows/app-build.yml